### PR TITLE
Fix the grid styles of the palettes

### PR DESCRIPTION
**Pull request by JonathanBytes**
# Baby, this is on fireeee ❤️‍🔥 PR

## Description

I fixed the grid layout on ImageGoWild and ImagIA pages. Please take a look at the deploy preview to ensure all is all right!

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

> I think we can do better for displaying the palettes on ImagIA (I don't complete like my design when the bg its the palette itself), but for now it solves the huge bug(feature?).

Have a great day! ❤️

### CODE REVIEW

**Comment by JonathanBytes** [comment]
I remove a lot of lines bc of duplicate code on the CSS.

File: src/ign-frontend/src/views/ImagIA.vue [hunk 1]
```diff
@@ -64,27 +63,27 @@ export default {
...
-  }
-}
+/* @media (min-width: 56.25em) { */
+/*   .palette-grid { */
+/*     .palette-post { */
+/*       width: 21%; */
+/*       max-height: 80px; */
+/*     } */
+/*   } */
+/* } */
```
Comment (TheJoin95) [inline src/ign-frontend/src/views/ImagIA.vue:86]: If not used, could you remove it from the codebase?

**Comment by TheJoin95** [comment]
About the rest, it seems a really solid PR. Thank you <3

**Comment by JonathanBytes** [comment]
It's a pleasure to contribute! :D

File: src/ign-frontend/src/views/ImagIA.vue [hunk 2]
```diff
@@ -4,9 +4,6 @@
         h1="Image Go AI 🤖"
         h2="Use the power of the PaletteNet to recolor your images"
       />
-    <center>
-      <a class="github-button" href="https://github.com/Schrodinger-Hat/ImageGoNord" data-color-scheme="no-preference: light; light: light; dark: dark;" data-size="large" data-show-count="true" aria-label="Star Schrodinger-Hat/ImageGoNord on GitHub">Star</a>
```
Comment (JonathanBytes) [inline src/ign-frontend/src/views/ImagIA.vue:8]: If you still want the widget at the top (Its at the bottom already) I can take a look why this is'nt working. But imo, is not necessary

Comment (TheJoin95) [reply src/ign-frontend/src/views/ImagIA.vue:8]: Let's take it as a new issue: I've added there to give more awareness to this repository as the goal is to get the project known here on GitHub :)

**Review by Wabri — APPROVED** [review]

**Pull request merged**